lib/theme/app_color_theme.dart

```diff
@@ -4,14 +4,15 @@ import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
 
 part 'app_color_theme.tailor.dart';
 
-@tailorMixinComponent
+@TailorMixinComponent()
```

Comment:
just curious, why this change :P ??

Reply:
I was curious if there is a difference between them :)

Reply:
i'm not an expert but the difference is that `@tailorMixinComponent` doesn't take any arguments and `@TailorMixinComponent( // can have some args specified here)` can specify some args with nondefault values. As i said, i've never educated myself on the topic but that's the difference in all these annotations i believe